$MirOS: ports/print/cups/patches/patch-cgi-bin_template_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- cgi-bin/template.c.orig	Tue Apr 25 17:15:22 2006
+++ cgi-bin/template.c	Tue May  9 17:39:25 2006
@@ -323,9 +323,9 @@ cgi_copy(FILE *out,			/* I - Output file
 	*/
 
         if (name[1])
-          sprintf(outval, "%d", cgiGetSize(name + 1));
+          snprintf(outval, sizeof(outval),"%d", cgiGetSize(name + 1));
 	else
-	  sprintf(outval, "%d", element + 1);
+	  snprintf(outval, sizeof(outval), "%d", element + 1);
 
         outptr = outval;
       }
